datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -8,7 +8,7 @@ boto3==1.10.27
 botocore==1.13.42
 clickhouse-cityhash==1.0.2.3
 clickhouse-driver==0.1.2
-contextlib2==0.5.5
+contextlib2==0.5.5; python_version < '3.0'
```

Comment:
Could you add a desc on the PR about why? That could be helpful :)

Reply:
Done!